packages/utils/src/EventHandlers/EventHandlers.ts

```diff
@@ -77,10 +77,11 @@ export abstract class EventHandlers<O extends Record<string, any> = {}> {
     return Object.keys(connectors).reduce(
       (accum, connectorName) => ({
         ...accum,
-        [connectorName]: (el, opts) => {
+        [connectorName]: (el, opts, ...args) => {
           this.registry.register(el, {
             opts,
             name: connectorName,
+            additionalArguments: args,
             connector: connectors[connectorName],
```

Comment:
Maybe we could change this a little. The first argument remains as `el` the DOM element, the second element becomes the required value, and then a 3rd argument for "options". So:

```suggestion
        [connectorName]: (el, required, opts) => {
          this.registry.register(el, {
            name: connectorName,
            required,
            opts,
            connector: connectors[connectorName],
```

This is related to #152 where we discussed the ability to add additional values as the 3rd parameter.

Reply:
Totally forgot about #152. Makes sense to make `opts` and object.